paltax/source_models_test.py

```diff
@@ -298,23 +306,36 @@ class CosmosCatalogTest(chex.TestCase, parameterized.TestCase):
     """Runs tests of CosmosCatalog functions."""
 
     def test__init__(self):
-        # Test that the intialization saves the path.
-        cosmos_catalog = source_models.CosmosCatalog(COSMOS_TEST_PATH)
-        self.assertEqual(cosmos_catalog.cosmos_path, COSMOS_TEST_PATH)
+        cosmos_catalog = source_models.CosmosCatalog(
+            COSMOS_TEST_PATH, images_per_chunk=3
+        )
+
+        # Test that the intialization saves the hdf5 file and the correct file is opened
+        self.assertEqual(cosmos_catalog.hdf5_file.filename, COSMOS_TEST_PATH)
+
+        # Test that the correct total number of galaxies is being stored (should be 5)
+        assert cosmos_catalog.total_num_galaxies == 5
```

Comment:
For consistency, can you use self.assertEqual here and in other places where you use assert